@here/harp-mapview/lib/ThemeLoader.ts

```diff
@@ -40,7 +40,7 @@ export interface ThemeLoadOptions {
     /**
      * Whether to resolve `ref` expressions in `definition` and `styles` elements.
      *
-     * @default `true`
+     * @default `false`, as featured datasources resolve definitions in [[StyleSetEvaluator]].
```

Comment:
I think the used of ``featured`` here is a bit confusing... I'd just say:
```suggestion
     * @default `false`, as datasources resolve definitions in [[StyleSetEvaluator]].
```

Reply:
applied